client/src/search/Typeahead/Typeahead.yaml

```diff
@@ -44,6 +23,6 @@ selected_result:
 menu_with_results_status:
   transform: [handlebars]
   en: |
-    {{total_matching_results}} results found. Showing results {{page_range_start}} to {{page_range_end}} of {{total_matching_results}}.
+    {{total_matching_results}} results found.
   fr: |
-    {{total_matching_results}} résultats trouvés. Affichage des résultats {{page_range_start}} à {{page_range_end}} sur {{total_matching_results}}.
+    {{total_matching_results}} résultats trouvés.
```

Comment:
One other super minor thing, haha, something about the period at the end of this bugged me. That just me?